Libs/RaptureCore/src/main/java/rapture/kernel/BlobApiImpl.java

```diff
@@ -125,14 +125,14 @@ public void deleteBlobRepo(CallingContext context, String blobRepoUri) {
                     }
                 }
             }
-            SearchPublisher.publishDeleteMessage(context, Kernel.getRepoCacheManager().getBlobConfig(uri.getAuthority()), uri);
         } catch (RaptureException e) {
             log.info("Unable to delete children; repo definitian may be invalid");
         }
 
         // delete parent directory
         BlobRepoConfigStorage.deleteByAddress(uri, context.getUser(), "Remove blob repo");
         removeRepoFromCache(uri.getAuthority());
+        SearchPublisher.publishDropMessage(context, blobRepoUri.toString());
```

Comment:
this wont work.  how did you test this?

  SearchPublisher.publishDropMessage calls SearchRepoUtils.getSearchRepo which has a case statement.  that case statement hasn't been updated.